src/lib/components/AssistantSettings.svelte

```diff
@@ -119,18 +121,29 @@
 			formData.set("ragAllowAll", "false");
 			formData.set("ragLinkList", "");
 			formData.set("ragDomainList", "");
+			formData.set("ragPrepromptUrl", "");
 		} else if (ragMode === "all") {
 			formData.set("ragAllowAll", "true");
 			formData.set("ragLinkList", "");
 			formData.set("ragDomainList", "");
+			formData.set("ragPrepromptUrl", "");
 		} else if (ragMode === "links") {
 			formData.set("ragAllowAll", "false");
 			formData.set("ragDomainList", "");
+			formData.set("ragPrepromptUrl", "");
 		} else if (ragMode === "domains") {
 			formData.set("ragAllowAll", "false");
 			formData.set("ragLinkList", "");
+			formData.set("ragPrepromptUrl", "");
+		} else if (ragMode === "preprompt") {
+			formData.set("ragAllowAll", "false");
+			formData.set("ragLinkList", "");
+			formData.set("ragDomainList", "");
+			formData.set(
+				"preprompt",
+				"The system prompt input is disabled when using a system prompt URL."
```

Comment:
We could just refactor later no? I don't get why we need this list this should be handled natively by the <form>.